src/hotspot/share/prims/jvmti.xml

```diff
@@ -3188,12 +3188,12 @@ err = (*jvmti)-&gt;Deallocate(jvmti, stack_info);
       </parameters>
       <errors>
         <error id="JVMTI_ERROR_OPAQUE_FRAME">
-          Attempted to return early from a frame
-          corresponding to a native method.
+          The implementation is unable to force current frame to return
+          (e.g. current frame is executing a native method).
+        </error>
+        <error id="JVMTI_ERROR_OPAQUE_FRAME">
```

Comment:
Do we still need this section? Same question for repeated sections below.

Reply:
Thank you for suggestion. Let's think on this a little bit.

Reply:
Thank you for the suggestion. Fixed now.